tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -187,6 +188,12 @@ sub find_sriov_ethernet_devices {
 sub enable_vf {
     my @pfs = @_;
 
+    #get br0 MAC for SUTs in OSD as the nic name may change after vfs are enabled
+    my $br0_mac = "";
+    if (script_run("ls /sys/class/net/br0/address") == 0) {
```

Comment:
Would be better to use:
```
my $br0_add = "/sys/class/net/br0/address";
if (script_run("[ -e $br0_add ]")) {
```

Reply:
Yes, better readability. Change to use '[ -e <file>]' but keep the variable($br0_add) definition as they are different.